### DISCUSSION TITLE
External USB drive. Not finding path

### DISCUSSION THREAD

**Discussion by phagerman** [TheLastGimbus/GooglePhotosTakeoutHelper#294] [I NEED HELP]
Hi -
Thanks for writing this script. It sounds like a lifesaver for me to get my photos out of Google Photos. I have 450gb of photos that have been unzipped into a single Takeout folder. I think I followed the instructions correctly (see screen shots). My issue is that I can't store all of those on my laptop drive so they are on an external USB drive connected to my Mac. I've tried every way that I know of to select the correct folders when I run the program but it always ends with "Input folder does not exist" (see screen shot). Are there any issues with the script knowing the path to an external usb drive on a mac? I'm guessing I'm doing something wrong but feel like I've tried everything I know. Thanks in advance for the help.

![Screenshot 2024-03-08 at 12 32 27 PM](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/94244620/1f78ac8e-fc3f-4640-8f52-b407abe8f03a)
![Screenshot 2024-03-08 at 12 32 19 PM](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/94244620/3127c17f-c4f1-435a-8003-20b763091414)
![Screenshot 2024-03-08 at 10 28 12 AM](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/94244620/0e9c46f9-58b6-413c-83e7-4017ca140fdf)

**Comment by TheLastGimbus** [comment]
Read stuff at https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/issues/251#issuecomment-1747006340
